### ISSUE TITLE
TypeError in reading self.bids_data['func_metadata']['PhaseEncodingDirection']

### ISSUE THREAD

**Issue by ericearl** [DCAN-Labs/abcd-hcp-pipeline#5]
The error is:

```
Traceback (most recent call last):
  File "/app/run.py", line 305, in <module>
    _cli()
  File "/app/run.py", line 65, in _cli
    return interface(**kwargs)
  File "/app/run.py", line 232, in interface
    session_spec = ParameterSettings(session, out_dir)
  File "/app/pipelines.py", line 133, in __init__
    self.bids_data['func_metadata']['PhaseEncodingDirection'])
TypeError: list indices must be integers or slices, not str
```

The `"PhaseEncodingDirection":` in each case in each `func` sidecar JSON is `"j"`.

I'm guessing from this the error is here:

https://github.com/DCAN-Labs/abcd-hcp-pipeline/blob/develop/app/helpers.py#L245

**Comment by ericearl** [comment]
Oh, and FYI, this is being tested with a docker built version of the develop branch on Rushmore with the "ANATFUNC" subject per your suggestion.

**Comment by dasturge** [comment]
addressed by 4f49aaf on develop
